src/test/rustdoc/intra-doc-link-generic-params.rs

```diff
@@ -0,0 +1,10 @@
+#![crate_name = "foo"]
+
+//! Here's a link to [`Vec<T>`] and one to [`Box<Vec<Option<T>>>`].
+//!
+//! And what about a link to just [`Option`] and, with the generic, [`Option<T>`]?
```

Comment:
```suggestion
//! And what about a link to just [`Option`] and, with the generic, [`Option<T>`]?
//! Don't forget [Vec::<T>::new]!
```
I think it will link to https://doc.rust-lang.org/nightly/alloc/vec/struct.Vec.html#method.new

Reply:
I already have a version of that locally, but I'm going to wait to commit it until I add support for it :)